pkgs/by-name/gl/glsl_analyzer/package.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, pkgs
+, fetchgit
+, zig_0_12
+, gitMinimal
+, darwin
+}:
+
+let stdenv = if pkgs.stdenv.isDarwin then darwin.apple_sdk_11_0.stdenv else pkgs.stdenv; in
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "glsl_analyzer";
+  version = "1.4.4";
+
+  src = fetchgit {
```

Comment:
Please use `fetchFromGitHub` instead. Also is `leaveDotGit` necessary?

Reply:
I've switched to `fetchFromGitHub`, but glsl_analyzer does not compile without `leaveDotGit` because it reads the commit hash for use in `glsl_analyzer --version`. This is also why `gitMinimal` is in `nativeBuildInputs`.